Pack-Smart/pack-smart-fe
# Styles/home page

## What's this PR do?
- Styles the home page with randomly generated travel images
- Create an images folder within the home component to hold the images and a `imageData.js` file
- Styles the header title container with an opacity 
- Adds margin to the packinglist header 
- Home page is fully responsive

## Where should the reviewer start?
- `Home.js`, `Home.scss` and images folder within Home

## How should this be manually tested?
- Start Pack Smart and refresh the home page to see different images appear
- View the home page on different screen sizes

## What are the relevant tickets?
- Closes #113 

## Screenshots/GIFs
<img width="1139" alt="Screen Shot 2021-03-02 at 1 00 03 PM" src="https://user-images.githubusercontent.com/58453566/109707408-5e560900-7b57-11eb-8eb9-c199567b6cdd.png"><img width="200" alt="Screen Shot 2021-03-02 at 1 00 16 PM" src="https://user-images.githubusercontent.com/58453566/109707417-601fcc80-7b57-11eb-87e5-af53bd82ea72.png">

## Did you update the README?
- Nope...but I will later today!
